.travis.yml

```diff
@@ -5,9 +5,9 @@ env:
 matrix:
   include:
     - python: "2.7"
-      env: TEST_REQUIRE="gevent greenlet geventhttpclient fastimport"
+      env: TEST_REQUIRE="gevent greenlet geventhttpclient fastimport mock"
```

Comment:
Please don't add extra dependencies. There should be no need for mocking here; you can just create a HttpGitClient and verify that the right attributes are set.